src/System.Private.CoreLib/shared/System/Globalization/TimeSpanFormat.cs

```diff
@@ -126,7 +127,7 @@ private static string FormatG(TimeSpan value, DateTimeFormatInfo dtfi, StandardF
 
         private enum StandardFormat { C, G, g }
 
-        private static bool TryFormatStandard(TimeSpan value, StandardFormat format, string decimalSeparator, Span<char> destination, out int charsWritten)
+        private static bool TryFormatStandard(TimeSpan value, StandardFormat format, string? decimalSeparator, Span<char> destination, out int charsWritten)
```

Comment:
Why can decimalSeparator be nullable?

Reply:
this was super complex logic in this function but essentially: whenever user passes anything else than format == StandardFormat.C we expect this to be non-null otherwise null (see assert I added 150 later (line 285) - still same function...)

Reply:
(also see example call on line 75)